components/match2/wikis/arenafps/legacy/match_maps_legacy.lua

```diff
@@ -0,0 +1,196 @@
+---
+-- @Liquipedia
+-- wiki=arenafps
+-- page=Module:MatchMaps/Legacy
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Arguments = require('Module:Arguments')
+local Array = require('Module:Array')
+local Json = require('Module:Json')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+local MatchGroup = require('Module:MatchGroup')
+local PageVariableNamespace = require('Module:PageVariableNamespace')
+local Table = require('Module:Table')
+
+local MatchGroupBase = Lua.import('Module:MatchGroup/Base')
+local MatchSubobjects = Lua.import('Module:Match/Subobjects')
+
+local globalVars = PageVariableNamespace()
+
+local OpponentLibraries = require('Module:OpponentLibraries')
+local Opponent = OpponentLibraries.Opponent
+
+local MAX_NUMBER_OF_OPPONENTS = 2
+local MAX_NUM_MAPS = 9
+local DRAW = 'draw'
+local SKIP = 'skip'
+
+local MatchMapsLegacy = {}
+
+---@param args table
+---@return table
+function MatchMapsLegacy._handleMaps(args)
+	Array.mapIndexes(function(index)
+		local prefix = 'map' .. index
+		local map = args[prefix]
+		local winner = Table.extract(args, prefix .. 'win')
+		local score = Table.extract(args, prefix .. 'score')
+		if Logic.isEmpty(map) and Logic.isEmpty(winner) then
+			return false
+		end
+
+		if Logic.isNotEmpty(score) then
+			local splitedScore = Array.parseCommaSeparatedString(score, '-')
+			args[prefix .. 'score1'] = mw.text.decode(splitedScore[1])
```

Comment:
Sometimes score input is `mapXscore=20-&#45;`

Reply:
Could we bot them?

Reply:
&#45; is `-`, Idk a good way to handle this, other than converting the match group

Reply:
Is the score `20&#45;10` or `20-&#45;10`? In the latter case it should just be removable with bot? And the former replaced by a normal dash?